docs/Chapter02-Overview-LoggingIn-and-Authentication.md

```diff
@@ -66,23 +62,25 @@ Most of us have hundreds of services that require a username and password. Most
 
 There are two popular password manager products that you should consider using: LastPass and 1Password. Both have similar features, but **LastPass** has a free tier with all the main features, and it can be used/installed without installing an application on your computer, so it is suggested for most users.
```

Comment:
Is this still true given shenanigans?

Reply:
Agree, good catch. Nixing LastPass here.